INFRASTRUCTURE.md

```diff
@@ -0,0 +1,77 @@
+# Our Infrastructure
+
+First, let's address some details of this document.
+
+<details>
+  <summary>What, why, who...</summary>
+
+  ## What?
+
+  This document outlines the various systems that JSON Schema the organisation uses and maintains.
+  This does not include social media profiles.
+
+  ## Why?
+
+  It's part of the required tasks to be completed by the OpenJS Foundation, forming an item of the [onboarding checklist](https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#onboaghrding-checklist).
+  It's also probably good practice.
+
+  This enables members of the JSON Schema team, and the OpenJS Foundation, to have oversight and management of such infrastructure should it be required.
+
+  ## Who?
+
+  This was mainly a list created by Ben Hutton (@relequestual).
+  Others are free to amend and update this (Via Pull Requests) as is required.
+
+</details>
+
+JSON Schema mostly operates on GitHub and Slack.
+
+Our GitHub organisation is located at https://github.com/orgs/json-schema-org
+
+Our Slack server is located at https://json-schema.slack.com
+(Anyone can join using https://json-schema.org/slack)
```

Comment:
```suggestion
Our public Slack workspace is https://json-schema.slack.com [Join](https://json-schema.org/slack)
```

Reply:
it's more I wanted to document that the joining link exists.